lib/phone_shop/screens/home_screen.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/phone_shop/widgets/components/fill_button.dart';
+import 'package:flutter_internship/phone_shop/widgets/components/shop_card.dart';
+import 'package:flutter_internship/phone_shop/widgets/data/shop_cards.dart';
+import 'package:flutter_svg/svg.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFFF8F8F8),
+      appBar: _HomeScreenTitle(),
+      body: SafeArea(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            _ShopCardList(),
+            _DisabledButton(),
```

Comment:
Buton shouldn't be fixed below List, it should be in front of List, like while you scrolling it should get behind the button